au4/src/app/app.cpp

```diff
@@ -60,7 +60,7 @@ App::App()
 
 void App::addModule(modularity::IModuleSetup* module)
 {
-    m_modules.push_back(module);
+    m_modules.emplace_back(module);
```

Comment:
The use of emplace_back does not matter here, because only a pointer is inserted, and not some structure by which memory is allocated and copied.